hipcub/PKGBUILD

```diff
@@ -2,46 +2,29 @@
 pkgname=hipcub
 pkgver=3.3.0
 pkgrel=2
-pkgdesc="Reusable software components for the rocm developers."
+pkgdesc='header-only library on top of rocPRIM or CUB'
```

Comment:
Can you explain why this description was changed from the official description: https://github.com/ROCmSoftwarePlatform/hipCUB

Reply:
I find the old description meaningless and wanted to change it to something more descriptive. It's basically the first sentence in the readme of hipcub.

Reply:
Sure

Reply:
Capitalize it, please. It looks better with starting capital letter.